diff --git a/oss/lib/cli_bridge.go b/oss/lib/cli_bridge.go
index 31a41af..4f9a262 100644
--- a/oss/lib/cli_bridge.go
+++ b/oss/lib/cli_bridge.go
@@ -193,11 +193,11 @@ func getSessionCredential(profile *config.Profile) (string, string, string, erro
 		return "", "", "", err
 	}
 	var lastErr error
-	accessKeyID, err := credential.GetAccessKeyID()
+	accessKeyID, err := credential.GetAccessKeyId()
 	if err != nil {
 		lastErr = err
 	}
-	accessSecret, err := credential.GetAccessSecret()
+	accessSecret, err := credential.GetAccessKeySecret()
 	if err != nil {
 		lastErr = err
 	}
diff --git a/oss/lib/ecs_role.go b/oss/lib/ecs_role.go
index 05fc3a9..e7fa658 100644
--- a/oss/lib/ecs_role.go
+++ b/oss/lib/ecs_role.go
@@ -36,7 +36,7 @@ type EcsRoleAK struct {
 	SecurityToken   string
 }
 
-func (ecsRole *EcsRoleAK) GetAccessKeyID() string {
+func (ecsRole *EcsRoleAK) GetAccessKeyId() string {
 	return ecsRole.AccessKeyId
 }
 
diff --git a/oss/lib/ecs_role_test.go b/oss/lib/ecs_role_test.go
index 666ba1d..6c997bb 100644
--- a/oss/lib/ecs_role_test.go
+++ b/oss/lib/ecs_role_test.go
@@ -112,14 +112,14 @@ func (s *OssutilCommandSuite) TestEcsRoleAkTimeout(c *C) {
 	time.Sleep(time.Duration(1) * time.Second)
 
 	ecsRole := EcsRoleAKBuild{url: "http://127.0.0.1:32915/latest/meta-data/Ram/security-credentials/EcsRamRoleTesting"}
-	strKeyId1 := ecsRole.GetCredentials().GetAccessKeyID()
+	strKeyId1 := ecsRole.GetCredentials().GetAccessKeyId()
 	c.Assert(strKeyId1 == "", Equals, false)
 	Expiration1 := ecsRole.Expiration
 
 	// wait Ak timeout
 	time.Sleep(time.Duration(1+TestEcsTimeout) * time.Second)
 
-	strKeyId2 := ecsRole.GetCredentials().GetAccessKeyID()
+	strKeyId2 := ecsRole.GetCredentials().GetAccessKeyId()
 	c.Assert(strKeyId2 == "", Equals, false)
 	Expiration2 := ecsRole.Expiration
 
