diff --git a/credentials/credential_test.go b/credentials/credential_test.go
index 73cb073..248358b 100644
--- a/credentials/credential_test.go
+++ b/credentials/credential_test.go
@@ -179,7 +179,7 @@ func Test_doaction(t *testing.T) {
 		Proxy: "# #%gfdf",
 	}
 	content, err = doAction(request, runtime)
-	assert.Equal(t, `parse # #%gfdf: invalid URL escape "%gf"`, err.Error())
+	assert.Equal(t, `parse "# #%gfdf": invalid URL escape "%gf"`, err.Error())
 	assert.NotNil(t, err)
 	assert.Nil(t, content)
 }
diff --git a/credentials/profile_provider_test.go b/credentials/profile_provider_test.go
index 7f5a044..9d4cf28 100644
--- a/credentials/profile_provider_test.go
+++ b/credentials/profile_provider_test.go
@@ -295,13 +295,13 @@ func TestProfileProvider(t *testing.T) {
 	p = newProfileProvider("NonExist")
 	c, err = p.resolve()
 	assert.Nil(t, c)
-	assert.Equal(t, "ERROR: Can not load section section \"NonExist\" does not exist", err.Error())
+	assert.Equal(t, "ERROR: Can not load section section 'NonExist' does not exist", err.Error())
 
 	// testcase 6, credential type does not set
 	p = newProfileProvider("notype")
 	c, err = p.resolve()
 	assert.Nil(t, c)
-	assert.Equal(t, "Missing required type option error when getting key of section \"notype\": key \"type\" not exists", err.Error())
+	assert.Equal(t, "Missing required type option error when getting key of section 'notype': key 'type' not exists", err.Error())
 
 	// testcase 7, normal AK
 	p = newProfileProvider()
